lib/lambda_ethereum_consensus/beacon/sync_blocks.ex

```diff
@@ -40,7 +40,8 @@ defmodule LambdaEthereumConsensus.Beacon.SyncBlocks do
 
   @spec perform_sync([chunk()]) :: :ok
   def perform_sync(chunks) do
-    Logger.info("[Optimistic Sync] Blocks remaining: #{Enum.count(chunks) * @blocks_per_chunk}")
+    remaining = chunks |> Stream.map(fn %{count: c} -> c end) |> Enum.sum()
```

Comment:
I don't see when this would make any difference?

Reply:
The last chunk has a different number of blocks, so approximating it by `Enum.count(chunks) * @blocks_per_chunk` rounds it up to the next multiple of 20.